machine-runner/src/runner/runner.ts

```diff
@@ -545,9 +545,28 @@ export type MachineRunnerIterableIterator<
 > = AsyncIterable<StateOpaque<SwarmProtocolName, MachineName, string, StateUnion>> &
   AsyncIterableIterator<StateOpaque<SwarmProtocolName, MachineName, string, StateUnion>> &
   AsyncIterator<StateOpaque<SwarmProtocolName, MachineName, string, StateUnion>, null> & {
+    /**
+     * @deprecated use `peekNext` instead
+     *
+     * The returned promise resolves when the machine-runner async iterator has a ready-state.
```

Comment:
```suggestion
     * The returned promise resolves when the machine-runner async iterator has received more events and computed the next ready-state.
```

Reply:
also below

Reply:
I’d also point towards `get()` for non-blocking state access, to make it explicit that this is blocking.